frontend/src/components/ViewActivities/index.test.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { render, screen, cleanup } from '@testing-library/react';
+import ViewActivities from './index.js';
+import authUtils from '../AuthUtils';
+
+const FAKE_USER = 'totally-legit-user';
+const FAKE_TRIPID = '12345';
+const RESULT_AUTHORIZED = FAKE_TRIPID;
+const RESULT_NOT_AUTHORIZED =
+    'Sorry, you\'re not authorized to view this trip.';
+const RESULT_TRIP_DOESNT_EXIST =
+    'Sorry, we couldn\'t find the trip you were looking for.';
+
```

Comment:
Just curious, is there a way to cover the case for when `isLoading` is true? I am not sure if this is possible because you would have to catch the component before it "mounts." But I thought I would ask in case you know this is possible.

Reply:
Yeah I'm not entirely sure how exactly to unit test for that besides mocking firestore completely to do nothing, which kind of defeats the point I think.

If anything, the loading component does render during manual tests.

Reply:
Okay sounds good.